### PR TITLE
Insticator Bid Adaptor: add support for different privacy platforms, ortb site first party data, fix video validation bug

### PR DESCRIPTION
**Pull request by shubhamc-ins**
## Type of change

- [x] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

**Comment by shubhamc-ins** [comment]
This PR contains fix for https://github.com/prebid/Prebid.js/issues/11224 issue as well.

**Comment by shubhamc-ins** [comment]
Hey @ChrisHuie @patmmccann. Please let us know when this can be released.

File: modules/insticatorBidAdapter.js [hunk 1]
```diff
@@ -190,31 +197,96 @@ function buildDevice(bidRequest) {
   return device;
 }
 
+function _getCoppa() {
+  return config.getConfig('coppa') === true ? 1 : 0;
```
Comment (ChrisHuie) [inline modules/insticatorBidAdapter.js:201]: Can you get this similar to other regs with 'ortb2.regs.coppa' instead please.

Comment (shubhamc-ins) [reply modules/insticatorBidAdapter.js]: @ChrisHuie Thanks!

Updated the PR. Let me know if everything looks good.

Comment (shubhamc-ins) [reply modules/insticatorBidAdapter.js]: The test case failing is related to some other prebid core updates I guess, in file `/test/spec/e2e/outstream/basic_outstream_video_ad.spec.js`.

@ChrisHuie

Comment (shubhamc-ins) [reply modules/insticatorBidAdapter.js:212]: @ChrisHuie Thanks!

Updated the PR. Let me know if everything looks good.

**Comment by shubhamc-ins** [comment]
Hey @ChrisHuie , Please let us know if these changes are going to merged with this week's release or next week, as I have a prebid documentation update PR pending as well and I need to mention the prebid supported version in there, your confirmation would be really helpful here.

Thanks a lot :)

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**